apps/src/code-studio/pd/form_components/button_list.jsx

```diff
@@ -92,15 +102,15 @@ const ButtonList = React.createClass({
         >
           {typeof answer === "object" ?
             <div>
-              <span style={{verticalAlign: 'top'}}>
-                {answerText}
+              <span style={styles.inputLabel}>
+                {answerText}{answerText.endsWith(":") ? "" : ":"}
```

Comment:
I'm not sure I love this. Why not allow for the flexibility?

Reply:
Fair point. The form I was working on had them all end in ":", but I suppose the options can just supply that. I'll remove it.